### PR TITLE
Added OutgoingIntegrationRequestsDefaultTimeout config setting

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/mattermost/pull/23805

### CODE REVIEW

File: source/configure/integrations-configuration-settings.rst [hunk 1]
```diff
@@ -117,14 +117,30 @@ Enable OAuth 2.0 service provider
...
++------------------------------------------------------------------------------------------------+
+
 .. config:setting:: integrate-enableusernameoverride
   :displayname: Enable integrations to override usernames (Integrations)
   :systemconsole: Integrations > Integration Management
   :configjson: .ServiceSettings.EnablePostUsernameOverride
   :environment: MM_SERVICESETTINGS_ENABLEPOSTUSERNAMEOVERRIDE
 
-  - **true**: Webhooks, slash commands, OAuth 2.0 apps, and other integrations such as `Zapier <https://docs.mattermost.com/integrations/zapier.html>`__, will be allowed to change the username they are posting as.
-  - **false**: **(Default)** Custom slash commands can only post as the username of the user who used the slash command.
```
Comment (hanzei) [inline source/configure/integrations-configuration-settings.rst:127]: Should the docs not make clear what the default is?

File: source/configure/integrations-configuration-settings.rst [hunk 2]
```diff
@@ -143,8 +159,8 @@ Enable integrations to override usernames
   :configjson: .ServiceSettings.EnablePostIconOverride
   :environment: MM_SERVICESETTINGS_ENABLEPOSTICONOVERRIDE
 
-  - **True**: Webhooks, slash commands, and other integrations, such as `Zapier <https://docs.mattermost.com/integrations/zapier.html>`__, will be allowed to change the profile picture they post with.
-  - **False**: Webhooks, slash commands, and OAuth 2.0 apps can only post with the profile picture of the account they were set up with.
+  - **true**: Webhooks, slash commands, OAuth 2.0 apps, and other integrations such as `Zapier <https://docs.mattermost.com/integrations/zapier.html>`__, will be allowed to change the username they are posting as.
+  - **false**: **(Default)** Custom slash commands can only post as the username of the user who used the slash command.
```
Comment (hanzei) [inline source/configure/integrations-configuration-settings.rst:163]: That is not true. This settings is about the profile picture.

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for documenting my PR, Carrie :+1:

File: source/configure/integrations-configuration-settings.rst (hunk 1 above)
Comment (cwarnermm) [reply source/configure/integrations-configuration-settings.rst:127]: Yes. There's 2 pieces of code here: code that drives what comes back when the user searches, and code for the page content itself. I saw some syntax issues in this section, attempted to clean them up, but didn't fix everything the first time around. Should be good now.

File: source/configure/integrations-configuration-settings.rst (hunk 2 above)
Comment (cwarnermm) [reply source/configure/integrations-configuration-settings.rst:163]: Thanks! I fixed this too in my latest commit.

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Pull request merged**